src/components/TextInputFocusable/index.js

```diff
@@ -40,6 +43,8 @@ const propTypes = {
 };
 
 const defaultProps = {
+    defaultValue: undefined,
```

Comment:
Aren't the `undefined` by default already? Maybe these should be `null` instead?

Reply:
Any reason why this can't be an empty String instead?

Reply:
That makes sense 👍 I replaced these with empty string.